src/components/portaldescription/OurTeamStyles.module.scss

```diff
@@ -0,0 +1,59 @@
+.teamLine_big,
+.teamLine_small {  
+  display: flex;
+  flex-direction: row;
```

Comment:
default value of felx-direction is row. you don`t change that property and you can del this row